data/brands/office/financial.json

```diff
@@ -4,6 +4,16 @@
     "exclude": {"generic": ["^financial$"]}
   },
   "items": [
+     {
+      "displayName": "World Finance",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "brand": "World Finance",
+        "brand:wikidata": "Q3569971",
+        "name": "World Finance",
+        "office": "financial"
```

Comment:
This looks more like `shop=money_lender` given that their primary product is personal loans.

Reply:
I've updated the category to money lender